auth.go

```diff
@@ -205,7 +205,7 @@ func (x *CentralStats) IncrementLogout(logger *log.Logger) {
 }
 
 type Auditor interface {
-	AuditUserAction(identity, clientIp, action string)
```

Comment:
Make action type an enum, even if it has only one type in it. Go doesn't have an enum type, but there's an idiomatic way to do enums.